include/tester/db/Database.hpp

```diff
@@ -138,29 +60,33 @@ class Database
         return result.extract();
     }
 
-    std::vector<bsoncxx::document::value> get_all_documents(const std::string& collection_name)
+    std::vector<bsoncxx::document::value> getAllDocuments(std::string collectionName)
```

Comment:
same here regarding the `static constexpr`

Reply:
done